### PR TITLE
Add udata technical posts

### PR DESCRIPTION
**Pull request by abulte**

### CODE REVIEW

File: pages/udata/3/udata-3-usage-metrics-explained.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+# udata 3 usage metrics explained
+
+Usage metrics are basically download counts and views counts for various objects (datasets, resources...). In udata 3 we heavily refactored the way it works.
+
+## Current architecture
+
+In udata 3, we introduced InfluxDB to act as an aggregator between Matomo (`udata-piwik`) and `udata`. This interface is handled by [udata-metrics](https://github.com/opendatateam/udata-metrics). The process is two stepped:
+1. udata periodically requests Matomo stats for a given time period for datasets, reuses, resources... ; tries to map it to database objects and then send a count for this time period to InfluxDB
+2. udata periodically request InfluxDB for an aggregated stats count overtime for each object in the database and enriches the `metrics` attributes with it
```
Comment (maudetes) [inline pages/udata/3/udata-3-usage-metrics-explained.md:18]:
```suggestion
2. udata periodically requests InfluxDB for an aggregated stats count overtime for each object in the database and enriches the `metrics` attributes with it
```

File: pages/udata/3/udata-3-usage-metrics-explained.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+This refactoring allowed us to be more precise on our stats count, making the connection with Matomo less brittle and getting rid of the `metrics` collection that was growing huge in our database. Plus we got rid of some technical debt. We also had in mind to use the InfluxDB database for more features: time-based statistics on udata, external dashboard...
+
+![](https://storage.gra.cloud.ovh.net/v1/AUTH_0f20d409cb2a4c9786c769e2edec0e06/imagespadincubateurnet/uploads/upload_477b262a4b85f2ff0e46e801f74a6b45.png)
```
Comment (maudetes) [inline pages/udata/3/udata-3-usage-metrics-explained.md:22]: Should we put this image in this repository?

File: pages/udata/summary.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: udata technical posts summary
+keywords:
+  - udata
+description: Technicals posts about udata
+---
+
```
Comment (maudetes) [inline pages/udata/summary.md:7]: Maybe we can link back to this summary in these new technical pages?

**Review by maudetes — COMMENTED** [review]
Thank you! :pray: 

Do you think we could create discussions on udata 3, metrics in udata 3 and udata 4 in github discussions ourselves? It could help the community to get started on their specific concerns.

**Comment by abulte** [comment]
@maudetes yes good idea for the discussions, can you create them and propose changes with the links? 😇

**Comment by maudetes** [comment]
Let's merge now and iterate after

**Pull request merged**